packages/terra-notification-dialog/src/NotificationDialog.scss

```diff
@@ -0,0 +1,99 @@
+:local {
+  .notification-dialog {
+    background: var(--notification-dialog-background, #fff);
+    border-radius: var(--notification-dialog-border-radius, 0.3125rem);
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+
+    @media screen and (max-width: 768px) {
+      width: calc(100% - 2 * var(--notification-dialog-width, 10px));
+    }
+  }
+
+  .notification-dialog-body {
+    background-color: var(--notification-dialog-body-color, #fff);
+    display: inline-flex;
+    flex: 1 1 auto;
+    flex-direction: initial;
+    height: 100%;
+    overflow: auto;
+    padding: var(--notification-dialog-body-padding, 1.25rem);
+    width: 100%;
+  }
+
+  .notification-dialog-body-text {
+    background-color: var(--notification-dialog-body-color, #fff);
+    font-size: var(--notification-dialog-body-font-size, 1.25rem);
+  }
+
+  .header-body {
+    background-color: var(--notification-dialog-header-color, #f9f9f9);
+    border-color: var(--notification-dialog-header-border-color, #dedfe0);
+    border-style: var(--notification-dialog-header-border-style, solid);
+    border-width: var(--notification-dialog-header-border-width, 0.0625rem);
+    display: 0 0 auto;
+    font-family: var(--notification-dialog-header-font-family, Helvetica);
```

Comment:
If the header font family doesn't need to change from the font family set on the HTML element, I'd remove this. Check with @neilpfeiffer to be sure.